__tests__/components/editor/PropertyTemplateOutline.test.js

```diff
@@ -75,73 +75,89 @@ describe('getLookupConfigItem module function', () => {
   })
 })
 
-describe('<PropertyTemplateOutline />', () => {
-  const propertyRtProps = {
-    propertyTemplate:
-      {
-        propertyLabel: "Test Schema name as a literal",
-        propertyURI: "http://schema.org/name"
-      }
+it('getLookupConfigItems() retrieves info from the lookup config', () => {
```

Comment:
moved this test and the hasValueTemplateRef() tests up -- they are their own exports, FBOw